app/src/main/java/com/example/footprnt/Map/Util/PostAdapter.java

```diff
@@ -20,7 +14,7 @@
 import android.widget.ImageView;
 import android.widget.TextView;
 
-import com.example.footprnt.Map.Util.MapUtil;
+import com.example.footprnt.Map.PostDetailActivity;
```

Comment:
I don't see PostDetailActivity being added to this file but just the import, is this import needed?